README.md

```diff
@@ -1,2 +1,2 @@
 # Auto-Sub
-The codebase that powers an autonomous submarine implementation created for MTE 380 
+The codebase that powers an autonomous submarine implementation created for MTE 380
```

Comment:
Please don't make trivial changes

Reply:
I actually have no idea where/when that happened. Must have been an accident.

Reply:
Please remove it from the PR